style.css

```diff
@@ -256,51 +251,43 @@ a {
 }
 
 .contact {
-  width: 375px;
   border: 2px solid #fff;
   border-top-left-radius: 75px;
   color: white;
   background-color: #6070ff;
-  background-image: url("./img/contact_form_mobile.png");
+  background-image: url("./img/contact_form_desktop.png");
   background-repeat: no-repeat;
-  background-size: cover;
   display: flex;
   flex-direction: column;
-  justify-content: center;
   left: 328px;
+  width: 100%;
+  grid-column: 1/-1;
+  justify-content: center;
+  background-position: left center;
+  background-size: contain;
 }
 
 .contact-div {
   display: flex;
   flex-direction: column;
   align-items: flex-start;
-  width: 375px;
-  height: 870px;
-}
-
-.div-content {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  width: 375px;
+  width: 447px;
   height: 870px;
 }
 
 .contact-h1 {
   margin-top: 103px;
-  font-family: 'Poppins', sans-serif;
+  font-family: "Poppins", sans-serif;
   font-style: normal;
-  font-size: 25px;
-  text-align: center;
-  align-items: center;
-  color: #fff;
-  padding-right: 20px;
+  font-weight: 700;
+  font-size: 36px;
+  line-height: 52px;
 }
 
```

Comment:
- [ ] Here in the contact section, kindly fix too the position of the form to be placed in the center 💯 

![image](https://user-images.githubusercontent.com/93630700/214734997-07ef32a7-9bf7-4583-9815-1e5ab251f61e.png)